### PR TITLE
BaSP-MR-74: create edit forms

### PR DESCRIPTION
**Pull request by CristianLotorto**
Create, add, edit forms done!

### CODE REVIEW

File: src/Components/Classes/index.js [hunk 1]
```diff
@@ -1,9 +1,113 @@
...
+    const data = await response.json();
+    console.log(data);
+    setClasses([...classes, JSON.parse(body.body)]);
+  };
+
+  const showForm = () => {
+    setTimeout(() => {
+      setShow(!show);
+    }, 100);
+  };
```
Comment (YhanahiRosenthal) [inline src/Components/Classes/index.js:56]: i think you should add a '0', 100 ms is very little

File: src/Components/Classes/index.js [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+    activity: '',
+    slots: ''
+  });
+
+  const getClasses = async () => {
+    const response = await fetch(`${process.env.REACT_APP_API_URL}/class`);
+    const data = await response.json();
+    setClasses(data.data);
+    console.log(data.data);
+  };
```
Comment (YhanahiRosenthal) [inline src/Components/Classes/index.js:27]: you should remove all console.log unneccesary

File: src/Components/Classes/Form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
+/* eslint-disable prettier/prettier */
```
Comment (NicoCagnina) [inline src/Components/Classes/Form/index.jsx:1]: remove this

File: src/Components/Classes/Form/index.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      slots: ''
+    });
+    updateToggle.setUpdateMode(false);
+    show();
+  };
+
+  const onChangeHour = (e) => {
+    klass.setKlass({
+      ...klass.klass,
+      // eslint-disable-next-line prettier/prettier
```
Comment (NicoCagnina) [inline src/Components/Classes/Form/index.jsx:38]: remove this

File: src/Components/Classes/Form/index.jsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    klass.setKlass({
+      ...klass.klass,
+      day: e.target.value
+    });
+  };
+
+  const onChangeTrainer = (e) => {
+    klass.setKlass({
+      ...klass.klass,
+      // eslint-disable-next-line prettier/prettier
```
Comment (NicoCagnina) [inline src/Components/Classes/Form/index.jsx:53]: remove this

File: src/Components/Classes/Table/index.jsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
+/* eslint-disable prettier/prettier */
```
Comment (NicoCagnina) [inline src/Components/Classes/Table/index.jsx:1]: remove this

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/Components/Classes/Table/index.jsx (hunk 6 above)
Comment (CristianLotorto) [reply src/Components/Classes/Table/index.jsx:1]: if i remove that line prettier formats the arrays in a way that eslint makes conflict

File: src/Components/Classes/Form/index.jsx (hunk 3 above)
Comment (CristianLotorto) [reply src/Components/Classes/Form/index.jsx:1]: if i remove that line prettier formats the arrays in a way that eslint makes conflict

File: src/Components/Classes/Form/index.jsx (hunk 4 above)
Comment (CristianLotorto) [reply src/Components/Classes/Form/index.jsx]: removed

File: src/Components/Classes/Form/index.jsx (hunk 5 above)
Comment (CristianLotorto) [reply src/Components/Classes/Form/index.jsx]: removed

**Review by YhanahiRosenthal — APPROVED** [review]

**Review by NicoCagnina — APPROVED** [review]

**Pull request merged**